### PR TITLE
Update Use page to link back to FAQ 

### PR DESCRIPTION
**Pull request by Poopooracoocoo**
And other changes to bring it in line with the FAQ

### CODE REVIEW

File: _layouts/use.html [hunk 1]
```diff
@@ -21,6 +21,11 @@
         into the mods folder.
     </strong>
 </p>
+<p>
+    It is recommended that you read the
+    <a href="https://fabricmc.net/wiki/faq:user">Frequently Asked Questions for users!</a>
+    Modding-friendly alternative launchers such as MultiMC are recommended over Minecraft Launcher.
```
Comment (modmuss50) [inline _layouts/use.html:27]: I don't especially want to push 3rd party launchers this hard, the offiical and reccomended way to install fabric using the offical launcher. Its the only launcher that can be fully trusted with users creds, only launcher I can easily provide support for/guarantee it works as expected. Most players are also used to the official launcher installing mods can be daunting so keeping change to a minimum is best. 

There is nothing wrong with the other launchers, I just dont feel comfortable recommending them over the official launcher.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: _layouts/use.html (hunk 1 above)
Comment (Poopooracoocoo) [reply _layouts/use.html]: >the offiical and reccomended way to install fabric using the offical launcher

Ah, multiple pages of the Wiki, specifically the [FAQ](https://fabricmc.net/wiki/faq:user) and [Installation Instructions](https://fabricmc.net/wiki/install), say that MultiMC is recommended so I thought it'd be consistent to have that there too. 

Should that recommendation be removed from the Wiki too?

I completely understand your concerns. Removed it.

Comment (Poopooracoocoo) [reply _layouts/use.html]: Confirmed on Discord: MultiMC is no longer the recommended way for users of Fabric.

File: _layouts/use.html [hunk 2]
```diff
@@ -14,13 +14,23 @@
...
+        Most mods will also require you to install Fabric API. Download it from
         <a href="https://www.curseforge.com/minecraft/mc-mods/fabric-api">
-            Fabric API
+            CurseForge
         </a>
-        into the mods folder.
+         or 
+        <a href="https://modrinth.com/mod/fabric-api">
+            Modrinth
+        </a>
```
Comment (modmuss50) [inline _layouts/use.html:24]: Again, id perfer to just have a Curseforge link on the website.

**Pull request closed without merging**